domains/docs.stefdp.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Stef-00012",
+        "email": "stefano.delprete.08@gmail.com"
+    },
+    "record": {
+        "CNAME": "c2d298f98c-hosting.gitbook.io"
```

Comment:
404 CNAME record.

Reply:
![image](https://github.com/is-a-dev/register/assets/81536172/90f8808b-853a-4599-b214-905bdb0fcf02)
gitbook only gives me that url, what should i do?